### PR TITLE
Add Photoview Gallery

### PR DESCRIPTION
**Pull request by hellvesper**
Thank you for your submission to the FreeNAS community plugins repository!

Please ensure the following guidelines are met when submitting a new Plugin.

1. Add the entry to INDEX in alphabetical order
2. Includes a new icon in the ./icon/ directory
3. Ensure the submitted icon file is a valid PNG that is 128x128 in size
4. Add new plugin cirrus task

### CODE REVIEW

File: photoview.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "packagesite": "https://pkg.freebsd.org/FreeBSD:13:amd64/latest",
+    "fingerprints": {
+        "iocage-plugins": [
+                {
+                "function": "sha256",
+                "fingerprint": "b0170035af3acc5f3f3ae1859dc717101b4e6c1d0a794ad554928ca0cbb2f438"
+            }
+        ]
+    },
+    "revision": "0"
```
Comment (sonicaj) [inline photoview.json:33]: It is an integer value i.e `0`

**Review by sonicaj — COMMENTED** [review]
Thank you @hellvesper for your contribution. Please add this to `INDEX` as well so it is discoverable by TrueNAS.

File: photoview.json (hunk 1 above)
Comment (hellvesper) [reply photoview.json]: Fixed, thank you!

**Review by sonicaj — APPROVED** [review]
Thank you @hellvesper for the changes!

**Pull request merged**